docs/releases.asciidoc

```diff
@@ -6,6 +6,89 @@
+This could lead to a race condition when database schema was updated and Debezium was still processing events with old schema structure.
+To mitigate the problem a new version of https://github.com/debezium/postgres-decoderbufs[Protocol Buffers decoder plugin] was introduced that passes additional type metadata with each event.
+The connector is backward compatible with the old decoder plugin (using the original approach) but we strongly recommend to upgrade it to the latest one as soon as possible.
+The race condition issue can still happen when primary key structure is changed for the table as this information is still obtained from JDBC metadata.
+* Hardcoded schema version overrides schema registry version https://issues.jboss.org/browse/DBZ-466[DBZ-466]; The schema version returned for CDC message values (schema type dbserver1.inventory.customers.Envelope) has changed. While always `1` was returned in earlier versions, the schema version as managed by the schema registry will be returned in case the Avro serializer/deserializer is used. `Null` will be returned when using the JSON serializer/deserializer. Note that the schema version is only set during Avro message serialization, i.e. an SMT applied on the source side will also get null when querying for the schema version, as SMTs will be applied before the serialization.
+
+=== New Features
+
+This release includes the following new features:
+
+* PostgreSQL connector should work on Amazon RDS and be able to use the available plugin https://issues.jboss.org/browse/DBZ-256[DBZ-256]
+* Build Debezium against Kafka 1.0.0 https://issues.jboss.org/browse/DBZ-432[DBZ-432]
+* Build Debezium images with Kafka 1.0.0 https://issues.jboss.org/browse/DBZ-433[DBZ-433]
+* Protobuf message should contain type modifiers https://issues.jboss.org/browse/DBZ-485[DBZ-485]
+* Protobuf message should contain optional flags https://issues.jboss.org/browse/DBZ-486[DBZ-486]
+* Better support for large append-only tables by making the snapshotting process restartable https://issues.jboss.org/browse/DBZ-349[DBZ-349]
+* Support new wal2json type specifiers https://issues.jboss.org/browse/DBZ-453[DBZ-453]
+* Optionally return raw value for unsupported column types https://issues.jboss.org/browse/DBZ-498[DBZ-498]
+* Provide Postgres example image for 0.7 https://issues.jboss.org/browse/DBZ-382[DBZ-382]
+* Create an automated build for Postgres example image in Docker Hub https://issues.jboss.org/browse/DBZ-383[DBZ-383]
+* Move configuration of ProtoBuf code generation to Postgres module https://issues.jboss.org/browse/DBZ-416[DBZ-416]
+* Provide MongoDB example image for Debezium 0.7 https://issues.jboss.org/browse/DBZ-451[DBZ-451]
+* Align 0.7 Docker files with 0.6 before release https://issues.jboss.org/browse/DBZ-463[DBZ-463]
```

Comment:
That's not exactly a new feature (also the next three below). Could you double check the issues and adjust their issue type accordingly. Also "Align 0.7 Docker files with 0.6 before release" should be changed into something more descriptive; suppose you're the reader of this log and need to make sense out of the descriptions :-)